<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Galvo Controller</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
      <property name="horizontalSpacing">
       <number>8</number>
      </property>
      <item row="3" column="0">
       <widget class="QLabel" name="opacityLabel">
        <property name="text">
         <string>Window Opacity</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>110</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
        <property name="title">
         <string>ROI/Line</string>
        </property>
        <widget class="QPushButton" name="lineButton">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>70</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/line.png</normaloff>images/line.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="roiButton">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>70</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/freehand.png</normaloff>images/freehand.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>91</width>
           <height>41</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Select Laser Buttons to toggle, or manually pulse lasers.&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Drag crosshair with right mousebutton.&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Draw and select ROIs with left mousebutton.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Hover over for instructions&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pointButton">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>70</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/point.png</normaloff>images/point.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Clear ROIs</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="3" colspan="2">
       <widget class="QGraphicsView" name="graphicsView">
        <property name="minimumSize">
         <size>
          <width>530</width>
          <height>0</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="transformationAnchor">
         <enum>QGraphicsView::AnchorViewCenter</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSlider" name="opacitySlider">
        <property name="minimumSize">
         <size>
          <width>355</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <number>25</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="closeButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>300</height>
         </size>
        </property>
        <property name="title">
         <string>Laser Control</string>
        </property>
        <widget class="QPushButton" name="laser1Button">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>130</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:checked, QPushButton:pressed {
	background-color: rgb(255, 0, 0);
}</string>
         </property>
         <property name="text">
          <string>Laser1</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="pulseButton">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>160</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Pulse</string>
         </property>
        </widget>
        <widget class="QPushButton" name="laser2Button">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>200</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:checked, QPushButton:pressed {
	background-color: rgb(255, 0, 0);
}</string>
         </property>
         <property name="text">
          <string>Laser2</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="pulse2Button">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>230</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Pulse</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Pulse time (ms):</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="pulseSpin">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>50</y>
           <width>71</width>
           <height>22</height>
          </rect>
         </property>
         <property name="suffix">
          <string> ms</string>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
        <widget class="QSlider" name="lineSepCounter">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>80</y>
           <width>91</width>
           <height>19</height>
          </rect>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>100</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>10</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="intervalTypeLabel">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>100</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Line Intervals</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGalvo">
    <property name="title">
     <string>Galvo</string>
    </property>
    <addaction name="actionCalibrate"/>
    <addaction name="actionReset"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="actionAutosave"/>
    <addaction name="separator"/>
    <addaction name="actionConfigure"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionShow_Help"/>
   </widget>
   <addaction name="menuGalvo"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport">
   <property name="text">
    <string>Import Settings</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export Settings</string>
   </property>
  </action>
  <action name="actionCalibrate">
   <property name="text">
    <string>Calibrate</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset Calibration</string>
   </property>
  </action>
  <action name="actionAutosave">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Autosave On Close</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Configure Ports</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>Rename Lasers</string>
   </property>
  </action>
  <action name="actionEditRaster">
   <property name="text">
    <string>Set Raster Pixel Shift</string>
   </property>
  </action>
  <action name="actionShow_Help">
   <property name="text">
    <string>Show Help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineSepCounter</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>593</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>434</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
